cves/kernel/CVE-2019-15921.yml

```diff
@@ -153,10 +160,14 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This vulnerability was reported and likely discovered by Hulk Robot.
+    I encountered no other information as to how this vulnerability was found
+    after looking through all the references in the CVE entry and searching the
+    internet for this bug and "Hulk Robot".
```

Comment:
Interesting. I was only able to find a dos testing utility over http which doesnt make sense here. Maybe try giving it another look. Seems strange they would mention something like that and you aren't able to find it anywhere. Maybe include the link where they mention it if you cant find anything

Reply:
Hulk robot is the name of the person who discovered it, not a tool.

Reply:
Yeah, it's a user who doesn't seem to have much online presence.